src/parser/github-comment-module.ts

```diff
@@ -279,6 +279,7 @@ export class GithubCommentModule implements Module {
           </h6>
         </b>
       </summary>
+      ${result.feeRate !== undefined ? `<h6>⚠️ ${new Decimal(result.feeRate).mul(100)}% fee rate has been applied.</h6>` : null}
```

Comment:
This is interesting. Curious to see this.

Reply:
It is in the linked QA